internal/loadbalancer/server.go

```diff
@@ -59,7 +59,11 @@ func StartLoadBalancerServer(props configuration.Configuration) {
 	// start server
 	log.Infof("Starting vedran load balancer on port :%d...", props.Port)
 	r := router.CreateNewApiRouter(*repos, props.WhitelistEnabled)
-	err = http.ListenAndServe(fmt.Sprintf(":%d", props.Port), r)
+	if props.CertFile != "" {
```

Comment:
What if someone only sets `--cert-file` flag, I guess this would fail?

I think this check should be made in `Args` function of `startCommand` as there we handle all flag validation.